docs/docs/installation/docker.md

```diff
@@ -0,0 +1,16 @@
+---
+sidebar_position: 2
+---
+
+# Docker
+## Installation
+If you have Docker installed, you can install TallyArbiter using our official Docker image [`TallyArbiter`](https://hub.docker.com/r/josephdadams/tallyarbiter).
+You can pull the image from DockerHub using the following command:
+```bash
+docker pull tallyarbiter:latest
+```
+
+Then, start it by typing `docker run -p 4455:4455 tallyarbiter`.
+TODO: explain mounting config.json
```

Comment:
We need to explain how to mount config.json to a file outside Docker container

Reply:
I don't use Docker at all so I have no idea how to do this.

Reply:
Let's provide a `docker run` command as well as a `docker-compose` example. I can add that tomorrow.

Reply:
Partially done. Need testing, because in the current env i can't use Docker.

Reply:
I don't think that $(pwd) works on Windows but I can check that later today. If it does not work, we could use an absolute path there so that the command stays cross-platform compatible?